Styles/style.css

```diff
@@ -170,22 +185,21 @@ body {
       transform: rotate(360deg);
     }
   }
-
   .sidenav {
     overflow-y: scroll;
```

Comment:
You don't have to specify the same property in media query if you haven't changed anything. You can remove it.